trinkets.lua

```diff
@@ -84,10 +84,10 @@ end
 function sArena.Trinkets:UNIT_SPELLCAST_SUCCEEDED(unitID, spell)
 	if not sArena.Trinkets[unitID] then return end
 	
-	if ( spell == GetSpellInfo(42292) or spell == GetSpellInfo(59752) )  then -- Trinket and EMFH
-		CooldownFrame_Set(sArena.Trinkets[unitID].Cooldown, GetTime(), 120, 1, true)
-	elseif ( spell == GetSpellInfo(7744) ) then -- WOTF
-		-- When WOTF is used, set cooldown timer to 30 seconds, but only if it's not already running or it has less than 30 seconds remaining
+	if ( spell == GetSpellInfo(195710) )  then -- Honorable Medallion
```

Comment:
I just updated the master branch with fixes for the trinkets so I don't think there's a need for this but please check the most recent update to see if I missed anything if you want.

Trinket icon change looks good.